erpnext/stock/report/itemwise_recommended_reorder_level/itemwise_recommended_reorder_level.py

```diff
@@ -55,14 +55,28 @@ def get_item_info(filters):
 
 
 def get_consumed_items(condition):
+	stock_entry_types_to_exclude = ', '.join([f"'{se}'" for se in [
+		"Material Transfer for Manufacture",
+		"Material Transfer",
+		"Send to Subcontractor"
+	]])
```

Comment:
Consider separating the list of purpose as a separate variable from query construction.